build_variables.bzl

```diff
@@ -562,6 +562,7 @@ torch_mobile_core = [
     "torch/csrc/jit/mobile/observer.cpp",
     "torch/csrc/jit/mobile/parse_bytecode.cpp",
     "torch/csrc/jit/mobile/parse_operators.cpp",
+    "torch/csrc/jit/mobile/quantization.cpp",
```

Comment:
What's the size effect of this file?

Reply:
It is quite small actually and also note that only here it is in mobile_core. In buckbuild.bzl i put it under training dep (no equivalent of that in build_variables.bzl), so in reality it should not have any impact on binary size.